src/main/java/co/cask/spanner/source/SpannerSource.java

```diff
@@ -71,6 +72,10 @@
   "SQL (ANSI 2011 with extensions), and automatic, synchronous replication for high availability.")
 public class SpannerSource extends BatchSource<NullWritable, ResultSet, StructuredRecord> {
   private static final Logger LOG = LoggerFactory.getLogger(SpannerSource.class);
+  private static final String TABLE_NAME = "TableName";
+  private static final Statement.Builder SCHEMA_STATEMENT_BUILDER =  Statement.newBuilder(
+    String.format("SELECT  t.column_name,t.spanner_type, t.is_nullable FROM information_schema.columns AS t WHERE " +
+                    "  t.table_catalog = ''  AND  t.table_schema = '' AND t.table_name = @%s", TABLE_NAME));
```

Comment:
this query is documented at https://cloud.google.com/spanner/docs/information-schema